src/app/pages/apps-old/docker-images/docker-images.store.ts

```diff
@@ -64,15 +64,15 @@ export class DockerImagesComponentStore extends ComponentStore<DockerImagesState
 
   readonly subscribeToUpdates = this.effect(() => {
     return this.ws.subscribe('core.get_jobs').pipe(
-      filter((event) => event.fields.method === 'container.image.pull' && event.fields.state === JobState.Success && !(event.msg === IncomingApiMessageType.Changed && event.cleared)),
+      filter((event) => event.fields.method === 'container.image.pull' && event.fields.state === JobState.Success && event.msg !== IncomingApiMessageType.Removed),
```

Comment:
We can make variables for each condition and give them descriptive names for better readability.